### PR TITLE
Minimize socket hangup errors from AWS

### PR DESCRIPTION
**Pull request by dora-korpar**
Still getting socket hangups on AWS tests, so this PR aims to mitigate them by increasing timeouts between AWS calls. It also changes the AWS test bucket used for the enterprise CI, as running the enterprise and public tests simultaneously on the same AWS bucket was causing interference.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -43,6 +43,18 @@ test:
     - mkdir -p $CIRCLE_TEST_REPORTS/unit
     - npm run unit_coverage
     - npm run multiple_backend_test
+    # Run S3 with multiple data backends ; run ft_awssdk
```
Comment (LaurenSpiegel) [inline circle.yml]: Is this file showing up because of a bad rebase?

File: tests/functional/aws-node-sdk/test/multipleBackend/put.js [hunk 2]
```diff
@@ -272,22 +255,14 @@ describe('MultipleBackend put object', function testSuite() {
...
-                                    'aws-test');
+                        setTimeout(() => {
+                            awsGetCheck(key, correctMD5, correctMD5, result => {
+                                assert.strictEqual(result.Metadata
+                                    ['x-amz-meta-unique-header'],
+                                    'second object');
                                 done();
                             });
-                        });
+                        }, 10000);
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/multipleBackend/put.js]: why longer for this one?

File: tests/functional/aws-node-sdk/test/multipleBackend/put.js [hunk 3]
```diff
@@ -372,14 +347,23 @@ describeSkipIfNotMultiple('MultipleBackend put object based on bucket location',
...
+                            assert.strictEqual(res.ETag, `"${correctMD5}"`);
+                            awsS3.getObject({ Bucket: awsBucket, Key: key },
+                            (err, res) => {
+                                assert.strictEqual(err, null,
+                                    `Expected success, got error ${err}`);
+                                assert.strictEqual(res.ETag, `"${correctMD5}"`);
+                                done();
+                            });
+                        });
+                    }, 10000);
```
Comment (LaurenSpiegel) [inline tests/functional/aws-node-sdk/test/multipleBackend/put.js]: and this one? let's use a constant defined at the top of the file for the delay time.

File: circle.yml (hunk 1 above)
Comment (dora-korpar) [inline circle.yml]: No, I moved the multiple backend tests up so they get run first so we can see if they fail right away.

**Comment by dora-korpar** [comment]
@ironman-machine go

**Comment by dora-korpar** [comment]
@ironman-machine go

**Comment by dora-korpar** [comment]
@ironman-machine go

**Pull request merged**